governance-services/latest/install/zip.md

```diff
@@ -6,12 +6,12 @@ Governance Services is installed by applying two AMP files to an existing Alfres
 
 The Governance Services distribution zip file contains the following files:
 
-|alfresco-governance-services-enterprise-repo-11.153.amp|Contains Governance Services functionality that's applied to an existing Alfresco Content Services installation.|
-|alfresco-governance-services-enterprise-share-11.77.amp|Contains Governance Services functionality that's applied to an existing Alfresco Share installation.|
+|alfresco-governance-services-enterprise-repo-17.173.amp|Contains Governance Services functionality that's applied to an existing Alfresco Content Services installation.|
+|alfresco-governance-services-enterprise-share-17.138.amp|Contains Governance Services functionality that's applied to an existing Alfresco Share installation.|
 
 > **Note:** Install the AMPs manually using the Module Management Tool (MMT), rather than using the `apply_amps` tool.
 
-1. Browse to the [Alfresco Nexus repository](https://nexus.alfresco.com/nexus/#nexus-search;gav~org.alfresco~alfresco-governance*~7.2.0~~) and download `alfresco-governance-services-enterprise-distribution-7.2.0.zip`, which contains the following files:
+1. Browse to the [Alfresco Nexus repository](https://nexus.alfresco.com/nexus/#nexus-search;gav~org.alfresco~alfresco-governance*~7.3.0~~){:target="_blank"} and download `alfresco-governance-services-enterprise-distribution-7.3.0.zip`.
```

Comment:
This shows me the Community ZIP in Nexus when I'm not logged in - is this the expected behaviour?

Reply:
Enterprise customers should access Hyland Community - Community users should go to Nexus (also added comment in PR #860 ). If you prefer, you can add a direct link to the Hyland Community release page for AGS Enterprise.

```suggestion
1. Browse to [Hyland Community](https://community.hyland.com/){:target="_blank"} and download `alfresco-governance-services-enterprise-distribution-7.3.0.zip`.
```